### PR TITLE
Add logging output when a POST is made

### PR DESCRIPTION
**Pull request by berezovskyi**
@likelion I can update the statements if you pick a better topic.

### CODE REVIEW

**Comment by likelion** [comment]
We should probably call the topic `lisp(oslc)`. And also make it more informative: what are we sending to where.

**Comment by berezovskyi** [comment]
Done! But I can't see any related output even with the new debug topic initialised:

image

I am guessing it's just me because I did not see them in the output even before. This is the output from `list_debug_topics.`:

```
lisp(oslc)                    false []
...
lisp                           true [user_error]
lisp                           true [user_error]
lisp                           true [user_error]
lisp                           true [user_error]
list(oslc)                     true [user_error]
```

**Comment by likelion** [comment]
You need to enable topic first, `debug(lisp(oslc)).`

**Comment by berezovskyi** [comment]
Yes, that's what I did! Twice actually. Once in the prolog fiel and once from the CLI

**Comment by likelion** [comment]
Looks like you enabled `list(oslc)`, not `lisp(oslc)`.

**Comment by berezovskyi** [comment]
I had a feeling it was me, thank you! Now everything work plus I have updated the log message.

File: lib/oslc_lisp.pl [hunk 1]
```diff
@@ -21,13 +21,17 @@
 :- multifile lisp:funct/3.
 
 lisp:funct(send, [IRI, URI], true) :- !,
+  debug(lisp(oslc), 'Sending a POST to  ~w', [URI]),
   oslc_client:post_resource(IRI, URI, []).
 
 lisp:funct(send, [IRI, URI, Options], true) :- !,
+  debug(lisp(oslc), 'Sending a POST to ~w', [URI]),
```
Comment (likelion) [inline lib/oslc_lisp.pl]: Shall we do this for `post_resource`:
`debug(lisp(oslc), 'POSTing resource [~w] to [~w]', [IRI, URI]),`

File: lib/oslc_lisp.pl [hunk 2]
```diff
@@ -21,13 +21,17 @@
...
   oslc_client:post_resource(IRI, URI, Options).
 
 lisp:funct(send_graph, [Graph, URI], true) :- !,
+  debug(lisp(oslc), 'Sending a POST to ~w', [URI]),
```
Comment (likelion) [inline lib/oslc_lisp.pl]: ...and this for `post_graph`:
`debug(lisp(oslc), 'POSTing graph [~w] to [~w]', [Graph, URI]),`

**Comment by berezovskyi** [comment]
Sure, I will do the updates!

**Comment by likelion** [comment]
Looks good!

**Pull request merged**